lib/mutant/mutator/node/block.rb

```diff
@@ -36,7 +36,32 @@ def mutate_body
           return unless body
           emit(body)
           emit_body_mutations
+
+          mutate_body_receiver
+        end
+
+        # Mutate method send in body scope of `send`
+        #
+        # @return [undefined]
+        #
+        # @api private
+        def mutate_body_receiver
+          return unless n_send?(body)
+
+          body_meta = AST::Meta::Send.new(body)
```

Comment:
If you follow this one, ignore my comments on `#send_meta`. and leave it as is for consistency.